lib/rake/build.rake

```diff
@@ -108,8 +108,13 @@ namespace :build do
           end
         end
 
-        HipChat.log 'Seeding <b>dashboard</b>...'
-        RakeUtils.rake 'seed:all'
+        # Allow developers to skip the time-consuming step of seeding the dashboard DB.
+        if (rack_env?(:development) && CDO.skip_seed_all)
+          HipChat.log 'Not seeding <b>dashboard</b> due to CDO.skip_seed_all...'
+        else
+          HipChat.log 'Seeding <b>dashboard</b>...'
```

Comment:
Maybe you can add a message here about modifying `locals.yml` if this is taking too long.